packages/redux/src/users/returns/selectors.ts

```diff
@@ -0,0 +1,94 @@
+import { createSelector } from 'reselect';
+import { getReturnItemsEntities, getReturnsEntities } from '../../returns';
+import { getUserReturns as getUserReturnsFromReducer } from '../reducer';
+import type { Return, ReturnItem } from '@farfetch/blackout-client';
+import type {
+  ReturnEntity,
+  ReturnEntityDenormalized,
+  ReturnItemEntity,
+  UserReturnsResultDenormalized,
+} from '../../entities';
+import type { StoreState } from '../../types/storeState.types';
+import type { UsersState } from '../types/state.types';
+
+/**
+ * Returns the loading status for the user returns operation.
+ *
+ * @param state - Application state.
+ *
+ * @returns User returns operation Loading status.
+ */
+export const areUserReturnsLoading = (state: StoreState) =>
+  getUserReturnsFromReducer(state.users as UsersState).isLoading;
+
+/**
+ * Returns the user returns error.
+ *
+ * @param state - Application state.
+ *
+ * @returns User returns operation error.
+ */
+export const getUserReturnsError = (state: StoreState) =>
+  getUserReturnsFromReducer(state.users as UsersState).error;
+
+const denormalizeReturnItem = (
+  returnItemId: ReturnItem['id'],
+  returnItems: Record<ReturnItemEntity['id'], ReturnItemEntity> | undefined,
+) => {
+  if (!returnItems) return;
```

Comment:
Prefer always this form for better readability as stated before. Change the other if below as well.

```suggestion
  if (!returnItems) { 
     return;
  }
```

Reply:
There is another `if` like this in line 92